railties/lib/rails/tasks/routes.rake

```diff
@@ -19,6 +19,9 @@ task routes: :environment do
 
   OptionParser.new do |opts|
     opts.banner = "Usage: rails routes [options]"
+
+    Rake.application.standard_rake_options.each { |args| opts.on(*args) }
```

Comment:
Hmm, come to think of it there might not be other cases since routes and test are the only rake tasks we fiddled with in Rails 5. So this feels like an ok compromise 👍